src/MCPClient/lib/clientScripts/extractContents.py

```diff
@@ -69,17 +74,17 @@ def main(transfer_uuid, sip_directory, date, task_uuid, delete=False):
         try:
             format_id = FileFormatVersion.objects.get(file_uuid=file_.uuid)
         # Can't do anything if the file wasn't identified in the previous step
-        except:
+        except BaseException:
```

Comment:
Catching `BaseException` is still very much a catch-all (it'll still mask things like name errors) and while this change keeps the linter from complaining I'm not sure if that's a good thing. For example, here we could catch `FileFormatVersion.DoesNotExist`.

There are several cases of this - deciding which exception types should be handled for each might be out of scope for this piece of work, but perhaps the linter should be allowed to keep note of this until they can be addressed more carefully.